src/Autocomplete/tests/Functional/AutocompleteFormRenderingTest.php

```diff
@@ -31,10 +31,17 @@ public function testFieldsRenderWithStimulusController()
             ->get('/test-form')
             ->assertElementAttributeContains('#product_category_autocomplete', 'data-controller', 'symfony--ux-autocomplete--autocomplete')
             ->assertElementAttributeContains('#product_category_autocomplete', 'data-symfony--ux-autocomplete--autocomplete-url-value', '/test/autocomplete/category_autocomplete_type')
+            ->assertElementAttributeContains('#product_category_autocomplete', 'required', 'required')
 
             ->assertElementAttributeContains('#product_portionSize', 'data-controller', 'symfony--ux-autocomplete--autocomplete')
+            ->assertElementAttributeContains('#product_portionSize', 'required', 'required')
+
+            // not implemented in zenstruck/browser
+            // ->elementAttributeNotExists('#product_colour', 'required')
```

Comment:
@kbond can i get at this assertion somehow?

Reply:
Try `->assertNotSeeElement('#product_colour[required]')`.

Reply:
works, thanks!